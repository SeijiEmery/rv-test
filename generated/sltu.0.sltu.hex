0000000    33  31  81  00  b3  b1  91  00  33  32  a2  00  b3  b2  b2  00
0000020    33  33  c3  00  b3  b3  d3  00  13  00  00  00  13  00  00  00
0000040    13  00  00  00  13  00  00  00  13  00  00  00  73  00  10  00
0000060
