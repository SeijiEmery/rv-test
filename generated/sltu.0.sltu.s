sltu x2, x2, x8
sltu x3, x3, x9
sltu x4, x4, x10
sltu x5, x5, x11
sltu x6, x6, x12
sltu x7, x7, x13
nop
nop
nop
nop
nop
ebreak
