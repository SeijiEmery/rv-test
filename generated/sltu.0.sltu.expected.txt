tests/sltu.test.s:1
R2 = 0
R3 = 1
R4 = 1
R5 = 1
R6 = 0
R7 = 0
